README.md

```diff
@@ -111,43 +108,24 @@ To find out the address of the RSS feed you want to subscribe to:
-| un document spécifique d'un service | Replace `$serviceId` with the service ID and `$documentType` with the document type:<br>`https://github.com/OpenTermsArchive/contrib-versions/commits/main/$serviceId/$documentType.md.atom` |
+| a specific document of a service | Replace `$serviceId` with the service ID and `$documentType` with the document type:<br>`https://github.com/OpenTermsArchive/contrib-versions/commits/main/$serviceId/$documentType.md.atom` |
 
 For example:
 
 - To receive all updates of `Facebook` documents, the URL is `https://github.com/OpenTermsArchive/contrib-versions/commits/main/Facebook.atom`.
 - To receive all updates of the `Privacy Policy` from `Google`, the URL is `https://github.com/OpenTermsArchive/contrib-versions/commits/main/Google/Privacy%20Policy.md.atom`.
 
-### Unsubscribe
-
-In order to not receive emails of updated services anymore, two links are included in every email received:
-
-- one to stop receiving all emails from bot@opentermsarchive.org
-- one to stop receiving emails of a particular document
-
-The latter consists in sending an email to contact@opentermsarchive.org to be manually removed from the corresponding list.
-## Contributing
-
-See [CONTRIBUTING](CONTRIBUTING.md).
-
-### Adding or updating a new service
-
-See the [CONTRIBUTING](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md) of repository [OpenTermsArchive/contrib-declarations](https://github.com/OpenTermsArchive/contrib-declarations).
-
 ## Using locally
 
-> **Windows Support**: This module can run locally on Windows systems.
-
 ### Installing
 
-- This module is built with [Node](https://nodejs.org/en/). You will need to [install Node >= v16.x](https://nodejs.org/en/download/) to run it.
-- When refering to the base folder, it means the folder where you will be `git pull`ing everything.
+This module is built with [Node](https://nodejs.org/en/) and is tested on macOS, UNIX and Windows. You will need to [install Node >= v16.x](https://nodejs.org/en/download/) to run it.
 
 #### Declarations repository
 
-1. Locally clone your declarations repository, e.g., `git@github.com:OpenTermsArchive/contrib-declarations.git`.
-2. Go into your folder and initialize it, e.g., `cd contrib-declarations; npm install`.
-3. You can now modify your declarations in the `./declarations/` folder, following [these instructions](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md).
+1. Locally clone your declarations repository, e.g., `git@github.com:OpenTermsArchive/template-declarations.git`.
+2. Go into your folder and initialize it, e.g., `cd template-declarations; npm install`.
+3. You can now modify your declarations in the `./declarations/` folder, following [these instructions](https://github.com/OpenTermsArchive/template-declarations/blob/main/CONTRIBUTING.md).
```

Comment:
You mentioned it was possible to not have the `blob/main` part of the url but I couldn't find how to.
If it's possible, please remove this part so that it works even if we switch branch (again)

Reply:
Unfortunately, I don't know how to do this either :(